src/routes/+page.svelte

```diff
@@ -31,26 +29,31 @@
 </main>
 
 <style>
+	* {
+		box-sizing: content-box;
+	}
 	main {
 		height: 100vh;
 		display: flex;
 		flex-direction: column;
+		background-image: linear-gradient(var(--navyblue), var(--lightbeige));
 	}
 	.upperHalf {
```

Comment:
Pretty sure you can archive the same effect by doing a linear gradient with a stop at 60 and then repeating the colors in oposite order

Reply:
cool, what would be the benefits of altering this?